src/picongpu/include/simulation_defines/param/synchrotronPhotons.param

```diff
@@ -36,6 +40,9 @@ BOOST_CONSTEXPR_OR_CONST float_64 SYNC_FUNCS_BESSEL_INTEGRAL_STEPWIDTH = 1.0e-3;
 /** Number of sampling points of the lookup table */
 BOOST_CONSTEXPR_OR_CONST uint32_t SYNC_FUNCS_NUM_SAMPLES = 8192;
 
-} // namespace bremsstrahlung
+/** Photons softer than this ratio: (2pi / omega) / DELTA_T, are neglected */
```

Comment:
This comment is not clear to me. What do you mean? `2 pi /  (omega DELTA_T)` has no unit. So you need to answer to what do I compare this unitless number 

Answer: if the frequency omega of your photon (`omega_phot`) is larger than `omega_phot > 2 pi * 1/(SOFT_PHOTONS_CUTOFF_RATIO * DELTA_T )` you create a photon.